src/08_comprehensions.py

```diff
@@ -11,14 +11,16 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
-
+for numbers in range(1, 6):
```

Comment:
Try to use comprehensions here and not loops. Also, check out the link in the file.